openlibrary/core/lending.py

```diff
@@ -300,11 +300,12 @@ def get_ocaid(ed):
 
     ocaids = [get_ocaid(ed) for ed in editions]
     ocaids = [ocaid for ocaid in ocaids if ocaid]
-    availabilities = get_availability_of_ocaids(ocaids)
-    for ed in editions:
-        ocaid = get_ocaid(ed)
-        success = ocaid and availabilities.get(ocaid)
-        ed['availability'] = availabilities.get(ocaid) if success else {'status': 'error'}
+    if ocaids:
```

Comment:
This just makes sure `ocaids` are not an empty list, otherwise ajax call will 404